src/tests/class_tests/openms/source/SwathMapMassCorrection_test.cpp

```diff
@@ -370,6 +370,8 @@ START_SECTION( void correctIM(const std::map<String, OpenMS::MRMFeatureFinderSco
   addTransitionsPep2(gr2, targ_exp);
   addTransitionsPep3(gr3, targ_exp);
 
+  const bool pasef = false;
```

Comment:
Definition of this variable is pretty far from its use. this makes the test harder to read.
alternatively you could just write:
```
mc.correctIM(transition_group_map, targ_exp, empty_swath_maps, false, im_trafo); // false = no pasef
```
making the calls easier to understand without scanning the code

Reply:
or if you want to have the constant to "name" parameters, maybe write:
```
const bool no_pasef{false}; // constant used to disable pasef option
```